cmd/verify/verify.go

```diff
@@ -26,7 +26,7 @@ type Message struct {
 	FileID       int    `json:"file_id"`
 	ArchivePath  string `json:"archive_path"`
 	FileChecksum string `json:"file_checksum"`
-	ReVerify     *bool   `json: "re_verify"`
+	ReVerify     *bool  `json: "re_verify"`
```

Comment:
```suggestion
	ReVerify     *bool  `json:"re_verify"`
```
This should make the tests pass.